spec/features/admin/admin_cannot_modify_users_spec.rb

```diff
@@ -1,3 +1,39 @@
+require 'rails_helper'
+
+RSpec.feature 'Admin can modify their account data' do
+  scenario 'Admin can not modify user data' do
+    admin = create(:user, role: 1)
+    user = create(:user)
+
+    visit login_path
+
+    fill_in 'Username', with: admin.username
+    fill_in 'Password', with: admin.password
+
+    click_on 'Save Session'
+
+    expect(page).to have_content('Modify Account Data')
+
+    click_on 'Modify Account Data'
+    expect(current_path).to eq(edit_user_path(admin))
```

Comment:
@NateAnderson1780 I think so. As it is, it tests the functionality fine, but it doesn't really test security. If some power-mad tyrant of an admin wanted to change user info, they might still be able to by entering /users/12/edit as a URL. (I think we have it set up so this isn't possible, but it would be good to have a test just to make sure)